### PR TITLE
fix: typo in comment

### PR DESCRIPTION
**Pull request by pengin7384**
**Description**

Fix typo in comment.

**Tests**

None

**Additional context**

None

**Metadata**

None

### CODE REVIEW

**Review by trianglesphere — APPROVED** [review]
TYVM

**Review by tynes — APPROVED** [review]

**Comment by trianglesphere** [comment]
@pengin7384 please merge in develop to make CI pass

File: .changeset/smooth-ducks-explode.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+'@eth-optimism/contracts-periphery': patch
```
Comment (trianglesphere) [inline .changeset/smooth-ducks-explode.md:2]: wrong branch?  Sorry it's not well documented, but we don't use changesets for golang code

Comment (pengin7384) [reply .changeset/smooth-ducks-explode.md:2]: So, should I revert this commit? What should I do after approval? Any help would be appreciated :)

Comment (trianglesphere) [reply .changeset/smooth-ducks-explode.md:2]: yea, revert it & then merge in the latest develop

Comment (pengin7384) [reply .changeset/smooth-ducks-explode.md]: Ok. I reverted my commit.

**Comment by pengin7384** [comment]
@trianglesphere How can I activate mergify? The merge condition seems to be met.

**Comment by trianglesphere** [comment]
> @trianglesphere How can I activate mergify? The merge condition seems to be met.

merge conditions are not met. You need to merge in the latest version of develop to make CI pass.

**Comment by pengin7384** [comment]
Thank you! :)

**Pull request merged**